### DISCUSSION TITLE
Twitter Cookie. How to configure

### DISCUSSION THREAD

**Discussion by weskerty** [DIYgod/RSSHub#16746] [Q&A]
I have copied the browser cookie with javascript:prompt("Cookies:", document.cookie);

But when I start the bot I get a configuration error message, Twitter API is missing.
How do I configure it correctly?

**Comment by slikie** [comment]
I have been using username and password for this

**Reply by weskerty** [reply]
That way it works the first time, then access is blocked.

**Comment by dzx-dzx** [comment]
The required cookie, `authCode`, is [`httpOnly`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#httponly) and must be copied from the devtool's Storage tab.
